### ISSUE TITLE
Consider removing the `exclude` directives in `go.mod`?

### ISSUE THREAD

**Issue by aofei** [anacrolix/torrent#689]
I suggest that this project should remove all `exclude` directives from the [go.mod](https://github.com/anacrolix/torrent/blob/c442a22ac2bb1d191cd5f77b496161ddc6fb6548/go.mod) file, because they prevent people from using the `go install` command.

```bash
$ go install github.com/anacrolix/torrent/cmd/...@latest
go install: github.com/anacrolix/torrent/cmd/...@latest (in github.com/anacrolix/torrent@v1.37.0):
        The go.mod file for the module providing named packages contains one or
        more exclude directives. It must not contain directives that would cause
        it to be interpreted differently than if it were the main module.
```

**Comment by anacrolix** [comment]
What version of go are you using?

**Comment by aofei** [comment]
This restriction actually existed when 'go install path@version' was first introduced (Go 1.16).

And I'm using Go 1.17.3.

**Comment by anacrolix** [comment]
There are versions of bazil.org/fuse that break the fs package after a certain version. Go doesn't provide a way to limit the version in use, or to fork bazil.org/fuse without manually rewriting all the import paths in that repo. I want to fix this, and prioritize having go install work externally over the excludes, but it's a shitty situation.

**Comment by anacrolix** [comment]
Thank you for reporting this! It's fixed in v1.38.0.
